cirq-core/cirq/protocols/json_test_data/CircuitOperation.json

```diff
@@ -205,6 +205,7 @@
           "cirq_type": "ParamResolver",
           "param_dict": []
         },
+        "parent_path": [],
```

Comment:
Is this JSON format backwards-compatible? i.e. can objects serialized in the old format still be deserialized in the new format? Full serialization guidelines for reference: [doc](https://github.com/quantumlib/Cirq/blob/master/docs/dev/serialization.md)

Same for the `MeasurementGate` JSON changes.

Reply:
Thanks for catching this. Moved old JSON serializations to `_inward` file and made the `parent_path` arg optional in `_from_json_dict_`.
`MeasurementGate` changes are backwards compatible. Not `_inward`-ing `MeasurementKey` JSON changes since no code or user serializes it to JSON yet.

Reply:
LGTM.